Tests/Common/test_segmentation_configs.py

```diff
@@ -212,7 +186,8 @@ def test_head_and_neck_paper_with_some_ground_truth_ids(
     Check that passing a num_structures between 1 and len(defaults) generates the correct subset.
     """
     ground_truth_ids = DEFAULT_HEAD_AND_NECK_GROUND_TRUTH_IDS[:ground_truth_count]
-    config = HeadAndNeckPaper(num_structures=ground_truth_count)
+    config = HeadAndNeckPaper(local_dataset=Path("foo"),
```

Comment:
Do you still need that extra constructor arg "local_dataset=...", now that you hardcoded the azure dataset ID in the model? (see my comment above - I think actually this change here is better than providing a fake Azure dataset ID)